lib/improver/tests/utilities/test_cube_checker.py

```diff
@@ -30,14 +30,20 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Unit tests for the cube_checker utility."""
 
-import numpy as np
 import unittest
 
 import iris
+import numpy as np
 from iris.cube import Cube
-from iris.tests import IrisTest
 from iris.exceptions import CoordinateNotFoundError
+from iris.tests import IrisTest
 
+from improver.tests.nbhood.nbhood.test_BaseNeighbourhoodProcessing import (
+    set_up_cube)
+from improver.tests.set_up_test_cubes import (
+    set_up_variable_cube, set_up_probability_cube)
```

Comment:
```suggestion
    set_up_variable_cube, 
    set_up_probability_cube)
```
As a general rule, this would be neater, as below

Reply:
Can be out of scope for this ;)